Project-1/README.md

```diff
@@ -12,63 +12,296 @@ Tip #4: The work we do is challenging; that should be assumed. You are smart eno
+  
   (a) ∃x∀y and ∀x∃y
+  Answer: the former means that there exists a x that for all y such that, while the latter means that for all x there exists a y such that.
+  
   (b) ∃x∀y∃z and ∀x∃y∀z 
+  Answer: the former means that there exists a x that for all y there exists a z such that, while the latter means that for all x there exists a y that for all z such that. 
+  
   (c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
-```
+  Answer: the former means that for all x, there exists a y that for all z, there exists a w such that, while the latter means that there exists a x that for all y, there exists a z that for all w such that.
+  
 	
 7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
-```
+
 ∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
 ∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
-```
+
+Answer: 
+
+The following graph models the second sentence but not the first sentence:
+Verticles: a, b, c
+Edges: ab, ac, aa (loop)
+
+See attached picture: image 0.
+
 	
 8. Using an online tableau proof generator - such as the one found here `https://www.umsu.de/trees/` - provide tree proofs of the following entailments, which are known as the De Morgan's laws:
-  ```(a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+  (a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+  Tree proof:
+  ¬∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+  ¬∀y(¬(Pa ∧ Qa) → (¬Pa ∨ ¬Qa))      (1)
+  ¬(¬(Pa ∧ Qa) → (¬Pa ∨ ¬Qa)).       (2)
+         ¬(Pa ∧ Qa)                  (3)
+         ¬(¬Pa ∨ ¬Qa)                (3)
+            ¬¬Pa                     (5)
+            ¬¬Qa                     (5)
+             Qa                      (7)
+             Pa                      (6)
+  
+  ¬Pa (4)              ¬Qa (4)
+   x                   x
+             
   (b) ∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
+  Tree proof:
+  
+   ¬∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))  
+   ¬∀y(¬(Pa ∨ Qa) → (¬Pa ∧ ¬Qa))     (1)
+    ¬(¬(Pa ∨ Qa) → (¬Pa ∧ ¬Qa))      (2)
+           ¬(Pa ∨ Qa)                (3)
+           ¬(¬Pa ∧ ¬Qa)              (3)
+	       ¬Pa                   (4)
+               ¬Qa                   (4)
+
+¬¬Pa (5)                ¬¬Qa (5)  
+  Pa (8)                  Qa (9)
+  x                       x
+  
   (c) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
-  (d) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
-```
+  Tree proof:
+  ¬∀x∀y((¬Px ∨ ¬Qx) →  ¬(Px ∧ Qx))
+   ¬∀y(¬(Pa ∨ Qa) →  ¬(Pa ∧ Qa))     (1)
+    ¬(¬(Pa ∨ Qa) →  ¬(Pa ∧ Qa))      (2)
+           ¬Pa ∨  ¬Qa               (3)
+           ¬¬(Pa ∧ Qa)              (3)
+	    Pa ∧ Qa                 (5)
+	       Pa                   (6)
+               Qa                   (6)
+
+  ¬Pa (4)                 ¬Qa (4)  
+  x                        x
+   
+  
+  (d) ∀x∀y(¬(Px ∨ Qx) → ¬(Px ∧ Qx))
+ Tree proof: same as (c)
+ 
 	
 9. Using a natural deduction proof generator - such as the one found here `https://proofs.openlogicproject.org/` - provide natural deduction proofs for each of De Morgan's laws. 
 
+
+  first step: (¬p ∨ ¬q) → ¬(p ∧ q)
+   
+ 1. ¬p ∨ ¬q          premise
```

Comment:
I asked you to prove quantified versions of Demorgan's Laws. You've attempted to prove propositional versions. You need to include the quantifiers.